.eslintrc-browser.json

```diff
@@ -0,0 +1,34 @@
+{
+  "root": true,
+
+  "extends": "jquery",
+
+  // Support:
+  //  - Desktop: Chrome 16+, Edge 12+, Firefox 3.6+, Internet Explorer 7+1, Opera 11.6+, Safari 4.0+
+  //  - Mobile: Android 2.3+, iOS 5.1+
+  // More info: https://github.com/jquery/sizzle/wiki#-browser-support
```

Comment:
This comment's purpose is not to list the supported browsers. This is a support comment which means it's supposed to explain why something was done in subsequent lines. Here, the goal is to explain why we're restricting the parser to `ecmaVersion: 3`. See the [jQuery comment](https://github.com/jquery/jquery/blob/3.4-stable/.eslintrc-browser.json#L6-L8) for comparison.

However, Sizzle supports a lot of ES3 browsers so I'd just remove that whole comment. In jQuery it was useful as since jQuery 2.0+ didn't support IE 8 & older one could think ES5 could be assumed which it couldn't. In Sizzle it's obvious only ES3 can be assumed.